package.json

```diff
@@ -10,8 +10,9 @@
     "build": "NODE_ENV=production yarn workspaces foreach --parallel --topological run build",
     "build:clean": "yarn workspaces foreach --exclude eds-tokens run build:clean",
     "create-component": "plop",
-    "create-releases": "lerna version --no-push --conventional-commits --message 'chore(release): %s'",
+    "create-releases": "yarn create-releases:interactive --conventional-commits",
     "create-releases:alpha": "yarn create-releases --preid alpha prerelease",
+    "create-releases:interactive": "lerna version --no-push --message 'chore(release): %s'",
```

Comment:
ooh that prompting is awesome! will this still update the `CHANGELOG` files automatically?

Reply:
Doh, I don't think it will. Good catch.

I'll settle instead of manually running things like
- `yarn create-releases minor`
- `yarn create-releases patch`

when needed.